tests/test_transformer/test_object_transformer.py

```diff
@@ -4,7 +4,11 @@
 import yaml
 from linkml_runtime import SchemaView
 from linkml_runtime.dumpers import yaml_dumper
-from linkml_runtime.linkml_model import ClassDefinition, SlotDefinition, SchemaDefinition
+from linkml_runtime.linkml_model import (
+    ClassDefinition,
+    SlotDefinition,
+    SchemaDefinition,
+)
```

Comment:
I ran black over the file after saving so the formatting would be consistent, hence there are some layout diffs